.tekton/create-ci-image-pipelinerun.yaml

```diff
@@ -122,9 +140,15 @@ spec:
             values: 
               -  "pull_request"
       - name: skopeo-copy
-        taskRef: 
-          kind: ClusterTask
-          name: skopeo-copy
+        taskRef:
+          resolver: cluster
+          params:
+            - name: kind
+              value: task
+            - name: name
+              value: skopeo-copy
+            - name: namespace
+              value: openshift-pipelines
         workspaces:
           - name: images-url
             workspace: images-url
```

Comment:
You need to update param names here... SOURCE_IMAGE_URL, DESTINATION_IMAGE_URL

Reply:
Done